great_expectations/expectations/expectation.py

```diff
@@ -1421,7 +1467,19 @@ def _get_augmented_library_metadata(self):
                 [key in keys for key in {"tags", "contributors"}]
             )
             has_no_forbidden_keys = all(
-                [key in {"tags", "contributors", "package"} for key in keys]
+                [
+                    key
+                    in {
+                        "tags",
+                        "contributors",
+                        "requirements",
+                        "package",
+                        "maturity",
+                        "has_full_test_suite",
+                        "manually_reviewed_code",
+                    }
+                    for key in keys
+                ]
```

Comment:
A nit but we could keep this as a generator instead of converting to a list right?